docs/changes.rst

```diff
@@ -36,6 +36,11 @@ Released: not yet
 
 **Enhancements:**
 
+* Add end-end test for indications where an OpenPegasus server in docker
+  container is the WBEM Server and pywbem defines the client that manages
```

Comment:
On "pywbem defines": That sounds strange to me. How about "pywbem is used as" or "pywbem is"?

Reply:
Changed to pywbem is both the client that creates indication subscriptions and the the indication listener